pkg/cmd/cli/backup/download.go

```diff
@@ -70,7 +71,18 @@ func (o *DownloadOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.DurationVar(&o.Timeout, "timeout", o.Timeout, "maximum time to wait to process download request")
 }
 
-func (o *DownloadOptions) Validate(c *cobra.Command, args []string) error {
+func (o *DownloadOptions) Validate(c *cobra.Command, args []string, f client.Factory) error {
+
+	veleroClient, err := f.Client()
+	cmd.CheckError(err)
+
+	switch {
```

Comment:
for this command we can remove the switch, and always try to get the backup named `o.Name`.  So L79-80, 84 can be removed.  The restore command had a switch their because it can take *either* a backup name or a schedule name, and we only wanted to check existence for whichever one of those was specified.